stories/index.js

```diff
@@ -300,3 +300,17 @@ storiesOf('ChipInput', module)
   .add('with "filled" variant full width', () => (
     <ChipInput defaultValue={['foo', 'bar']} variant='filled' label='Add Tags' fullWidth fullWidthInput />
   ))
+  .add('tips for CJK IME user', () => (
+    <div>
+      <ChipInput
+        defaultValue={['猫', '犬']}
+        newChipKeys={[]}
+        onChange={action('onChange')}
+      />
+      <div>
+        add <code>{"newChipKeys={[]}"}</code> to property.
+        default is <code>{"newChipKeys={['Enter']}"}</code>.
+        prevent onChange event when IME is processing key input.
```

Comment:
How do we prevent this?

Reply:
when event.keyCode `13`, IME is not processing key input.
when `229`, IME is processing key input.
both situations have `event.key === 'Enter'`. so `newChipKeys={[]}` is prevent event when IME is processing key input.

My english isn't as good as Japanese, so I'm glad if you can fix it so that it can be understandable. 😺